bmx280: Unable to compile due to spi3w_en parameter

Hi RIOT members,

When I included bmx module in my project:
```
#include "bmx280.h"
#include <bmx280_params.h>
```

 I received :
`./domotique/RIOT/drivers/bmx280/include/bmx280_params.h:94:1: error: missing initializer for member 'bmx280_params_t::spi3w_en' [-Werror=missing-field-initializers]`

after a search on all RIOT I found the declaration but this 'spi3w_en' is never used... So I proposed to remove it ?
https://github.com/RIOT-OS/RIOT/blob/82ab75c2b4cf4deaa2bd630a3af63f1299c3fd48/drivers/include/bmx280.h#L198

after removing this line I can compile without issue.

Best regards,
Memiks.

--- a/drivers/include/bmx280.h
+++ b/drivers/include/bmx280.h
@@ -195,7 +195,6 @@ typedef struct {
     /* Config Register */
     bmx280_t_sb_t t_sb;                 /**< standby */
     bmx280_filter_t filter;             /**< filter coefficient */
-    uint8_t spi3w_en;                   /**< Enables 3-wire SPI interface */
 
     /* ctrl_meas */
     bmx280_mode_t run_mode;             /**< ctrl_meas mode */

